### PR TITLE
Clarify meaning of approval

### PR DESCRIPTION
**Pull request by JoelQ**
Based on a conversation that happened in Slack on what "approval" means
in the context of code review. This makes it explicit that the PR author
gets to make the final call on whether to merge or make changes. The
reviewer is there to give feedback, not to act as a gatekeeper.

Ideally, discussion and edits go on until both parties are satisfied. In
the event that this drags on too long or no agreement can be reached,
the author may act unilaterally to merge without making some of the
suggested changes.

### CODE REVIEW

**Comment by JoelQ** [comment]
Personally, I never use the "Request changes" option because it feels so negative (big red x and all that). I prefer to use either "Comment" (when I make a lot of suggestions or when I want to imply that I'd like to do a second round of review once changes are addressed) or "Approve" (when I don't expect to return to this PR).

Not sure how others use the system though 😄

File: code-review/README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Having Your Code Reviewed
 * Wait to merge the branch until continuous integration (TDDium, Travis CI,
   CircleCI, etc.) tells you the test suite is green in the branch.
 * Merge once you feel confident in the code and its impact on the project.
+* Final editorial control rests with the PR author.
```
Comment (jferris) [inline code-review/README.md]: Worth expanding out "PR" to "pull request" so it's easy to understand without more context?

(I was very tempted to use "Request Changes" for this comment)

Comment (JoelQ) [reply code-review/README.md]: good point

Comment (JoelQ) [reply code-review/README.md]: Fixed in d3542b5

**Comment by mike-burns** [comment]
LGTM, that's also how I use those buttons, and for history tracking: the prior discussion on this was in #279.

**Comment by JoelQ** [comment]
Good find @mike-burns ! 🕵️

**Pull request merged**